lib/common/schemas.c

```diff
@@ -926,8 +926,19 @@ cib_upgrade_err(void *ctx, const char *fmt, ...)
     va_end(ap);
 }
 
+/*!
+ * \internal
+ * \brief Apply a single XSL transformation to given XML
+ *
+ * \param[in] xml        XML to transform
+ * \param[in] transform  XSL name
+ * \param[in] to_logs    Something to do with logging
```

Comment:
If true, this causes non-parsing and non-validating error messages to pass through cib_upgrade_err (a function which is just completely out of control) at crm_log_level.